src/prefect/tasks/jupyter/jupyter.py

```diff
@@ -0,0 +1,66 @@
+import nbconvert
+import nbformat
+import papermill as pm
+
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class JupyterTask(Task):
+    """
+    Task for running Jupyter Notebooks.
+    In order to parametrize the notebook, you need to mark the parameters cell as described in
+        the papermill documentation: https://papermill.readthedocs.io/en/latest/usage-parameterize.html
+
+    Args:
+        - notebook_path (string, optional): path to fetch the notebook from.
+            Can be a cloud storage path.
+            Can also be provided post-initialization by calling this task instance
+        - notebook_params (dict, optional): dictionary of parameters to use for the notebook
+            Can also be provided at runtime
+        - as_html (bool, optional): whether to output as HTML. (default: True)
```

Comment:
Instead of using `as_html`, perhaps make this a string category so other nbconvert formats could be used (markdown, latex, etc...)? For now you could only support the two you currently support (json and html), but it leaves an opportunity for expansion later where a boolean flag doesn't. Additionally, I think the default should be `json`, as that allows more flexibility for downstream tasks to postprocess the results if needed (and most users are familiar with executed notebooks as json, while fewer may be familiar with them as html).


```python
output_type="json"
```